scripts/add_translation.py

```diff
@@ -71,18 +80,47 @@ def get_credentials():
     return credentials
 
 
+def display_languages(service):
+    print 'List of available languages:\n----------------------------'
+    data = service.files().list(q="'" + TRANSLATION_FOLDER_ID + "'" + " in parents").execute()
+    for f in data['files']:
+        if f['mimeType'].endswith('.folder'):
+            LANGUAGE_LIST[f['name']] = f['id']
+
+    # sort the languages alphabetically
+    sorted_list = [e for e in sorted(LANGUAGE_LIST)]
+    id = 1
+    # store the generated IDs to match against user input.
+    language_id = {}
+    for i in sorted_list:
+        print  str(id) + ' ' + i
+        language_id[id] = i
+        id = id + 1
```

Comment:
This line can be removed by using `enumerate`.

If it weren't for that, I'd point out that like many languages, python offers a `+=` operator as convenient shorthand. :)